app/models/fax_recipient.rb

```diff
@@ -0,0 +1,49 @@
+class FaxRecipient
+  attr_accessor :residential_address
+
+  def initialize(residential_address:)
+    @residential_address = residential_address
+  end
+
+  def number
+    Office.find_by(zip: residential_address.zip)["fax_number"]
+  end
+
+  def name
+    ENV.fetch("FAX_RECIPIENT", "Michigan Bridges")
+  end
+
+  class Office
```

Comment:
Generally prefer one class per file -- thoughts on putting this baby class in its own file?